genpack.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
+
+# generate `.tar.gz` packages from git repos
+
+# set PACKAGEURL
+set_pack_url() {
+	case "$1" in
+	tunasync)	PACKAGEURL=https://github.com/sjtug/tunasync.git ;;
```

Comment:
Use direct package download instead of keeping a local git repo? But why not use wget directly in Dockerfile...